tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Java.cs

```diff
@@ -198,5 +203,21 @@ public override Task CleanupAsync(string project)
 
             return Task.CompletedTask;
         }
+
+        private IDictionary<string, string> LoadSourceVersions()
+        {
+            var sourceVersions = new Dictionary<string, string>();
+            foreach (var line in File.ReadAllLines(VersionFile))
+            {
+                var trimmedLine = line.Trim();
+                if (string.IsNullOrEmpty(trimmedLine)) continue;
+                if (trimmedLine.StartsWith('#') || trimmedLine.StartsWith("beta_") || trimmedLine.StartsWith("unreleased_")) continue;
```

Comment:
```suggestion
                if (string.IsNullOrEmpty(trimmedLine) ||
                    trimmedLine.StartsWith('#') ||
                    trimmedLine.StartsWith("beta_", StringComparison.Ordinal) ||
                    trimmedLine.StartsWith("unreleased_", StringComparison.Ordinal))
                {
                    continue;
                }
```

Reply:
Or you could use `StringComparison.OrdinalIgnoreCase` if you want to match any case.